vectordraw.js

```diff
@@ -305,17 +362,24 @@ VectorDraw.prototype.updateVectorProperties = function(vector) {
         x2 = vector.point2.X(),
         y2 = vector.point2.Y();
     var length = length_factor * Math.sqrt(Math.pow(x2-x1, 2) + Math.pow(y2-y1, 2));
+    var slope = (y2-y1)/(x2-x1);
     var angle = ((Math.atan2(y2-y1, x2-x1)/Math.PI*180) - base_angle) % 360;
     if (angle < 0) {
         angle += 360;
     }
     $('.vector-prop-name .value', this.element).html(vector.point2.name); // labels are stored as point2 names
-    $('.vector-prop-length .value', this.element).html(length.toFixed(2) + ' ' + length_units);
-    $('.vector-prop-angle .value', this.element).html(angle.toFixed(2));
+    if (vector.elType !== "line") {
+        $('.vector-prop-length .value', this.element).html(length.toFixed(2) + ' ' + length_units);
+        $('.vector-prop-angle .value', this.element).html(angle.toFixed(2));
+    }
+    else {
+        $('.vector-prop-length', this.element).hide();
+        $('.vector-prop-angle', this.element).hide();
```

Comment:
@smarnach lines have angle too. Also, it ignores `show_slope_for_lines` setting - either restore it or remove from documentation